puppeteer-scraper/README.md

```diff
@@ -468,13 +474,15 @@ Since the input UI is fixed, it does not support adding of other fields that may
 With the final three options in the **Advanced configuration**, you can set custom names for the following:
 
 -   Dataset
+    -   Leave your dataset unnamed if you only want the data within it to be persisted on the Apify platform for 7 days (after which, it will expire). Named datasets are retained indefinitely. Learn more [here](https://docs.apify.com/storage#named-and-unnamed-storages).
 -   Key-value store
--   Request queue
+    -   Similarly to datasets, named key-value stores never expire, and unnamed ones expire after 7 days.
```

Comment:
While the infinite data retention is a feature of the named storages as well, the most important use-case for using a named storage here is to share it across multiple web scraper runs. E.g. instead of having 10 datasets for 10 runs over 10 days, you can have 1 dataset where all the data will accumulate over time.